spec/inquisition/i18n_tasks/runner_spec.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.describe Inquisition::I18nTasks::Runner do
+  describe '#call' do
+    subject(:runner) { described_class.new }
+
+    let(:base_task_instance) { instance_double(I18n::Tasks::BaseTask) }
+    let(:missing_keys_error) { { 'test.key' => instance_data_for_error } }
+    let(:instance_data_for_error) do
+      instance_double('I18n::Tasks::Data::Tree::Node', value: 'test', data: { path: 'test_path' })
+    end
+
+    before do
+      allow(I18n::Tasks::BaseTask).to receive(:new) { base_task_instance }
+      allow(base_task_instance).to receive(:unused_keys).and_return({})
```

Comment:
Would be better to return some of object, because you don't cover case with `unused_keys`